boring-sys/src/lib.rs

```diff
@@ -95,7 +76,7 @@ pub fn init() {
 
     let init_options = OPENSSL_INIT_LOAD_SSL_STRINGS;
 
-    INIT.call_once(|| unsafe {
-        OPENSSL_init_ssl(init_options, ptr::null_mut());
-    })
+    INIT.call_once(|| {
+        assert!(unsafe { OPENSSL_init_ssl(init_options.try_into().unwrap(), ptr::null_mut()) } == 1)
```

Comment:
Nit:  `assert_eq!` will be less error prone

Reply:
Sure!